apps/web/vibes/soul/lib/streamable.tsx

```diff
@@ -56,7 +56,7 @@ const promiseCache = weakRefCache<string, Promise<unknown>>();
  * A suspense-friendly upgrade to `Promise.all`, guarantees stability of
  * the returned promise instance if passed an identical set of inputs.
  */
-export function all<T extends readonly unknown[] | []>(
+function all<T extends readonly unknown[] | []>(
```

Comment:
Remove leftover `export`. This function is now exported as [`Streamable.all`](https://github.com/makeswift/vibes/pull/458/files#diff-683feafeea4c0466f0f41cb70a118638b750e435559cd80a4568c21848e73c66R76).